### DISCUSSION TITLE
What a about a "continue sailing" button at the menu bar panel?

### DISCUSSION THREAD

**Discussion by brad12majors** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1001] [Ideas]
Sometimes (eg. after a reboot) my macbook leaves sailing mode and starts to charge for 80% again. 
What about a button in the menu bar panel, that simply continues sailing mode?

Juergen

**Comment by MatthiasKerbl** [comment]
Hi Juergen,

We will release an update soon for the Sailing Mode soon which will allow to continue sailing after a reboot.

Best,
Matthias
